lib/action_subscriber/configuration.rb

```diff
@@ -68,10 +68,10 @@ def self.configure_from_yaml_and_cli(cli_options = {}, reload = false)
           yaml_config = {}
           absolute_config_path = ::File.expand_path(::File.join("config", "action_subscriber.yml"))
           if ::File.exists?(absolute_config_path)
-            erb = ::ERB.new(::File.read(absolute_config_path)).result
+            erb_yaml = ::ERB.new(::File.read(absolute_config_path)).result
             # Defined in Psych 3.2+ and the new canonical way to load trusted documents:
             # https://github.com/ruby/psych/issues/533#issuecomment-1019363688
-            yaml_config = ::YAML.unsafe_load(erb)[env]
+            yaml_config = ::YAML.respond_to?(:unsafe_load) ? ::YAML.unsafe_load(erb_yaml)[env] : ::YAML.load(erb_yaml)[env]
```

Comment:
is this safe for all psych versions? Should we have a more loose minimum constraint?

Reply:
That is the most permissive as it uses the new method provided in 3.3.2+, and falls back to <3.1 behavior otherwise. Specs for this are passing under 2.6, 2.7, 9.2, 9.3, 3.0, and 3.1 (though other specs fail in 3+). The default in 2.6, and 2.7 is 3.1, and the default in 9.3 is 3.3.2. I think we're good.